bin/plugin/lib/milestone.js

```diff
@@ -0,0 +1,96 @@
+/** @typedef {import('@octokit/rest')} GitHub */
+/** @typedef {import('@octokit/rest').IssuesListForRepoResponseItem} IssuesListForRepoResponseItem */
+/** @typedef {import('@octokit/rest').IssuesListMilestonesForRepoResponseItem} OktokitIssuesListMilestonesForRepoResponseItem */
+
+/**
+ * @typedef {"open"|"closed"|"all"} IssueState
+ */
+
+/**
+ * Given a SemVer-formatted version string, returns an assumed milestone title
+ * associated with that version.
+ *
+ * @see https://semver.org/
+ *
+ * @param {string} version Version string.
+ *
+ * @return {string} Milestone title.
+ */
+function getVersionMilestoneTitle( version ) {
+	const [ major, minor ] = version.split( '.' );
+
+	return `Gutenberg ${ major }.${ minor }`;
```

Comment:
Seems like this should be part of the config, the idea behind these folders is:

- "lib" should be config agnostic (pure functions where everything come from arguments)
- "commands" can use "config" but not hard code "Gutenberg" specific things. Today, this rule is not respected entirely  in all commands. Release command do respect it though.

Reply:
Hm. In that model, would it be simplest to move this function in its entirety to the command file? To your point, it could still leverage the configuration. Though it'd be a bit odd in how it assumes a certain convention of "plugin name" + " " (space) + WordPress-standard-version version number.

Maybe it's fine enough.

e.g. `${ config.name } ${ major }.${ minor }`

Reply:
yes, or having something like `%name %major %minor` in the config some kind of template. It does seem like this function might not be that useful in lib though.

Reply:
> yes, or having something like `%name %major %minor` in the config some kind of template.

I was initially inclined to avoid this since it would require something capable of performing the template substitution. I forgot that Node ships [a `util.format` utility](https://nodejs.org/api/util.html#util_util_format_format_args), though it doesn't allow for "naming" of the template parts to allow rearrangement. [`@wordpress/i18n`'s `sprintf`](https://github.com/WordPress/gutenberg/blob/master/packages/i18n/README.md#sprintf) (or `sprintf-js` directly) is another option as well, and does support named substitution.

Reply:
Updated in ac75a27 to extract a `versionMilestoneFormat` configuration.

Notably, this utility is also required by changelog, so I couldn't simply move it to the release script. But I tried to find a simple compromise that could be used by both. There's a bit of duplication, but it seems acceptable enough.